tests/unit/test_base_client.py

```diff
@@ -240,6 +258,161 @@ def _reraise_token_error(_: GlobusApp, error: TokenValidationError):
     assert str(ex.value) == "No token data for transfer.api.globus.org"
 
 
+class GlobusAppConfigurator:
```

Comment:
🤩 What a name!

Reply:
GlobusAppConfigConfig just didn't have the right ring to it.